package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fp-ts",
-  "version": "1.0.0-rc",
+  "version": "1.0.1",
```

Comment:
Please, avoid changing `package-lock`

Reply:
😱 oh no. i didnt mean to change that. I apologize.